net-im/signal-desktop-bin/signal-desktop-bin-1.32.1.ebuild

```diff
@@ -71,6 +71,9 @@ src_install() {
 pkg_postinst() {
 	xdg_desktop_database_update
 	xdg_icon_cache_update
+
+	elog 'When using Xfce, install  xfce-extra/xfce4-statusnotifier-plugin to '
+	elog 'be able to use the tray icon.'
```

Comment:
I think it'd be better if you made something like this:
```
if has_version "xfce-base/xfce4-session"; then
	optfeature "xfce-extra/xfce4-statusnotifier-plugin" tray icon in XFce desktop environment.
fi
```

Could you give your opinion? Even without `has_version` `optfeature` from *eutils.eclass* might be better here.

Reply:
Good suggestion. Implemented (but without `has_version`)